desktop/Automattic/wp-desktop/.circleci/config.yml

```diff
@@ -0,0 +1,342 @@
+version: 2.1
```

Comment:
Note: this publish-only config does not execute e2e tests, as we can rely on the test runs from within the Calypso repository.